templates/app.yaml

```diff
@@ -246,6 +246,13 @@ Resources:
         - Namespace: 'aws:elasticbeanstalk:application:environment'
           OptionName: TEAM_TO_ROLE_ARN_MAP
           Value: "ssm::/service-catalog/TeamToRoleArnMap"
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
```

Comment:
we don't need this, the ImportValue already provides  MARKETPLACE_ID_DYNAMO_TABLE_NAME  value to the login app.

Reply:
No sure what you are asking me to delete, just line 249?

Reply:
please ignore my comment here.  looks good.